src/common/helpers/utils.ts

```diff
@@ -171,6 +171,12 @@ export function getEntityLink(entity: {type: string, bbid: string}): string {
 	return `/${kebabCase(entity.type)}/${entity.bbid}`;
 }
 
+export function snakeCaseToSentenceCase(str) {
+	const camelCaseString = camelCase(str);
+	const wordsArray = camelCaseString.split(/(?=[A-Z])/);
+	const sentenceCaseString = wordsArray.map(word => upperFirst(word)).join(' ');
+	return sentenceCaseString;
+}
```

Comment:
So......I actually needed this for statements related to `Edition Group`. For example: The heading in RelationshipTypeTree component has the format "EntityType1-EntityType2 relationship types". For this, I was faced with the dilemma of what should I use for EditionGroups: should be called edition_group, Editiongroup, EditionGroup or Edition Group. I eventually settled with the sentenceCase because that's what musicbrainz [does](https://musicbrainz.org/relationships/release_group-release_group). 

Do you think this is reasonable?

Reply:
Oh, I understand the reason why you want the current result, and I agree!

However my question was about the implementation.
In the context of these entity names (which is all the code is used for IIUC), I don't see anything that your custom function does that lodash's `startCase` utility doesn't:
<img width="313" alt="image" src="https://github.com/metabrainz/bookbrainz-site/assets/6179856/1985b65e-9d06-42be-8ccb-4f5aeed8b6b6">

I'm just suggesting to drop this custom `snakeCaseToSentenceCase`  and use `startCase` instead.
IIUC start case and sentence case are the same thing (at least where our entity names are concerned…)

Reply:
You're right!